tests/test_distributions.py

```diff
@@ -25,7 +25,7 @@
     'l': '{"name": "LogUniformDistribution", "attributes": {"low": 0.001, "high": 100}}',
     'du': '{"name": "DiscreteUniformDistribution",'
     '"attributes": {"low": 1.0, "high": 10.0, "q": 2.0}}',
-    'iu': '{"name": "IntUniformDistribution", "attributes": {"low": 1, "high": 10}}',
+    'iu': '{"name": "IntUniformDistribution", "attributes": {"low": 1, "high": 10, "q": 1}}',
```

Comment:
The JSON-serialized distribution is stored in RDB storage without q argument. So we need to keep compatibility that restores IntUniformDistribution from `'{"name": "IntUniformDistribution", "attributes": {"low": 1, "high": 10}}'`.

I think the changes at [here](https://github.com/optuna/optuna/pull/910/files/85599e61e35ed5c7717dfc17f7c292e84db3dbf8#diff-5fabefd6403ba111713a236e60435579) made broken compatibilities fixed. So could you keep previous test cases?